### DISCUSSION TITLE
Soundboard?

### DISCUSSION THREAD

**Discussion by christomitov** [stoatchat/discussions#653] [Feature Suggestions]
Don't see this in the roadmap, how hard would it be to add soundboard feature?

**Comment by s-macinnis** [comment]
yes! this would probably make us switch our communities over.

**Comment by vicebooster** [comment]
This is such a neat idea!

**Comment by ghost** [comment]
> @christomitov: Don't see this in the roadmap, how hard would it be to add soundboard feature?
It would probably be easy, you can just send a WebSocket event

**Comment by sageelliott1509-del** [comment]
agreed

**Comment by Endorsed1320** [comment]
This!!
